src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
 public class WorkWithFile {
```

Comment:
Minor comment. 
Map is really heavy data structure, so there are no purposes to use it in this task, but it's great that you know it. 
Also, inner try-catch block - is not best practice
Please, try not to complicate your solution.